$OpenBSD: patch-include_x_h,v 1.1 2013/09/02 06:15:34 dcoppa Exp $

commit 103ab83131d30614ea9337e752548a834773bfc6
Author: Michael Stapelberg <michael@stapelberg.de>
Date:   Tue Aug 27 20:21:11 2013 +0200

Revert "raise fullscreen windows on top of all other X11 windows"

This reverts commit 0659a0d98ca98fcd134c277c6a61ca3c9a04544e.

It was a fix for dzen2 setups, but broke several other use cases.
Because dzen2 is not really important since most people use i3bar
nowadays, let's revert this.

--- include/x.h.orig	Wed Aug  7 20:50:24 2013
+++ include/x.h	Mon Sep  2 07:42:58 2013
@@ -93,12 +93,8 @@ void x_push_changes(Con *con);
  * Raises the specified container in the internal stack of X windows. The
  * next call to x_push_changes() will make the change visible in X11.
  *
- * If above_all is true, the X11 window will be raised to the top
- * of the stack. This should only be used for precisely one fullscreen
- * window per output.
- *
  */
-void x_raise_con(Con *con, bool above_all);
+void x_raise_con(Con *con);
 
 /**
  * Sets the WM_NAME property (so, no UTF8, but used only for debugging anyways)
